save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt

```diff
@@ -69,6 +70,32 @@ internal class OrganizationController(
         organizationService.findByName(organizationName)?.avatar.let { ImageInfo(it) }
     }
 
+    /**
+     * @param organizationName name of an organization
+     * @param newCanCreateContests new value of flag Organization.canCreateContests
+     * @param authentication an [Authentication] representing an authenticated request
+     * @return response
+     */
+    @PostMapping("/{organizationName}/manage-contest-permission")
+    @Suppress("UnsafeCallOnNullableType")
+    fun changeCanCreateContests(
```

Comment:
`changeCanCreateContests` -> `setAbilityToCreateContest`

Reply:
Will do it

Reply:
Done